.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/lutzroeder/Netron.git
 [submodule "Samples/RustSqueezenet/winrt-rs"]
 	path = Samples/RustSqueezenet/winrt-rs
-	url = https://github.com/microsoft/winrt-rs.git
+	url = https://github.com/microsoft/winrt-rs.git
```

Comment:
is there something different with your line ending settings? seems like this file (and at least one other) shouldn't have changes in it.